ansible/roles/debops.ferm/tasks/main.yml

```diff
@@ -23,7 +25,7 @@
     state: 'directory'
     owner: 'root'
     group: 'adm'
-    mode: '2750'
+    mode: '02750'
```

Comment:
This should probably have one 0 less :)

Reply:
This is a fix for the E202 error in `ansible-lint`. The `file` Ansible module seems to work as expected with this leading zero.

Reply:
Hm, but is the leading 0 needed. It looks wired. In all the other places we use 4 digits. My point here is consistency in the code base.

Reply:
I like to use 4 digits for the mode to be explicit so that it is clear that I did not forget a special permission. Interestingly the only use of 5 digits currently in master is apparently from me in the `debops.apt_cacher_ng` role (`ag 'mode:[^0-9\n]*[2-7][0-7]{3}'`).

The mode parameters with 4 digits not starting with 0 currently in master (that you fix in this PR), it has not caused issues because they are all quoted. I am still in favor of your point that you want to avoid permission issues. This only stands against that we have not seen/used this before DebOps really.

Thanks for explaining. Lets go with 5 digits in case the special file/dir permissions are used.

Using `ag 'mode:[^0-9]*[2-7][0-7]{3}'` I found one additional instance of this in `ansible/roles/debops.icinga_web/tasks/main.yml` which was not found by `ansible-lint`. Can you update this also?

Reply:
@ypid Thanks, done.